README.md

```diff
@@ -400,6 +402,16 @@ Required for [Docker](#docker) projects.
 
 Required for [balena](#balena) projects.
 
+#### `PYPI_TOKEN`
+[API key](https://pypi.org/help/#apitoken) for pushing releases to PyPI applications.
+
+Used for [python](#python) projects.
+
+#### `PYPI_TEST_TOKEN`
+[API key](https://pypi.org/help/#apitoken) for pushing releases to test PyPI applications.
```

Comment:
missing newline for Markdown syntax (try a markdown lint vscode extension)

Reply:
Done